Beam Quest Blogpost (#27004)
apache/beam: website/www/site/content/en/blog/beamquest.md

Individuals aren’t the only ones who can benefit from completing this quest -  organizations can too! Because earning this badge represents deep knowledge of an industry leading big data library, having the badge validates your organization’s understanding of Beam. In addition, you can run the Beam library on a wide variety of runners, including Google Cloud Dataflow, Flink, Spark, and more, making knowledge about this library highly transferable. Finally, your organization can use this quest as onboarding material for new hires on big data teams, allowing teams and organizations to get their newest employees up-to-date on the latest and greatest that Apache Beam has to offer.

> Given the recent advancements in artificial intelligence, now’s the time to jump into the world of data processing! Get started on your journey [here](https://www.cloudskillsboost.google/quests/310).

Reviewer: Could you add "Data processing is a key part of the AI and ML workflows." as the first sentence here?

---
title:  "Getting started with Apache Beam: An open source proficiency credential sponsored by Google Cloud"
date:   2023-06-09 00:00:01 -0800
categories:
  - blog
aliases:
  - /blog/2023/06/09/beam-quest.html
authors:
  - svetaksundhar
---
<!--
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
-->

<img class="center-block"
     src="/images/blog/apch-beam-w_bdg_c_en.png"
     alt="Quest image">

> The [Getting Started with Apache Beam quest](https://www.cloudskillsboost.google/quests/310) is a series of four labs that teach and test proficiency in Apache Beam. Upon completion, you're granted a sharable credential badge!

Author: do we need these two sentences here?

Reviewer: I agree. Let's remove this one.

Author: Done. Moved the quest link and removed link from "Google Cloud Badge" (bc its the same link)

---

> title:  "Get Started with Apache Beam: An Open Source Quest sponsored by Google Cloud"

Reviewer 2: ```suggestion
title:  "Get started with Apache Beam: An open source quest sponsored by Google Cloud"
```

> title:  "Getting started with Apache Beam: An open source quest sponsored by Google Cloud"

Reviewer: let's use a better SEO word here, like "credential" or "badge". 

(I feel like, developers would not search for "beam quests")

date:   2023-06-09 00:00:01 -0800
categories:
  - blog
aliases:
  - /blog/2023/06/09/beam-quest.html
authors:
  - svetaksundhar
---
<!--
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
-->

> ### **What is the quest?**

Reviewer: I think we can remove both headers.

Keep the first sentence.
Add a <!--more--> tag
Continue with the rest of the blog post.

> [Get Started with Apache Beam Quest](https://www.cloudskillsboost.google/quests/310) is a series of four qwiklabs that test proficiency in Apache Beam. Upon completion, a sharable credential badge is granted!

Reviewer 2: ```suggestion
The [Get Started with Apache Beam quest](https://www.cloudskillsboost.google/quests/310) is a series of four labs that teach and test proficiency in Apache Beam. Upon completion, you're granted a sharable credential badge!
```

> ### **Apache Beam Playground Features**

Reviewer 2: ```suggestion
### **Apache Beam playground features**
```

Author: this wasn't related to my change -- should I fix anyway?

Reviewer 2: You don't need to change it.

> We’re excited to announce the release of the “Get Started with Apache Beam” quest, a series of four online labs that venture into different Apache Beam concepts. When you complete all four labs, you’ll earn a [Google Cloud badge](https://www.cloudskillsboost.google/quests/310) that you can share on platforms like LinkedIn. Earning this badge should take less than seven hours total, and signing up for the quest costs $20 (however there are often free specials for people who attend Beam events, such as [Meetups](https://www.meetup.com/topics/apache-beam/), [Beam Summit](https://beamsummit.org/), and [Beam College](https://beamcollege.dev/)).

Reviewer 2: ```suggestion
We’re excited to announce the release of the “Get Started with Apache Beam” quest, a series of four online labs that venture into different Apache Beam concepts. When you complete all four labs, you’ll earn a [Google Cloud badge](https://www.cloudskillsboost.google/quests/310) that you can share on platforms like LinkedIn. Earning this badge should take less than seven hours total, and signing up for the quest costs $20 (there are often free specials for people who attend Beam events, such as [Meetups](https://www.meetup.com/topics/apache-beam/), [Beam Summit](https://beamsummit.org/), and [Beam College](https://beamcollege.dev/)).
```

> Beam is one of the largest big data open source projects actively being developed. Over the past six years, the Apache Beam community has seen tremendous growth in the number  of contributors, committers, and users. If you’re a long time Beam user, you can now earn a badge to show your skills to potential employers. If you’re new to Beam, you can begin your learning journey with this quest. To attempt this quest, you don’t need any prior knowledge of data processing or distributed systems. All you need is elementary knowledge about programming.

Reviewer 2: ```suggestion
Beam is one of the largest big data open source projects actively in development. Over the past six years, the Apache Beam community has seen tremendous growth in the number  of contributors, committers, and users. If you’re a long time Beam user, you can now earn a badge to show your skills to potential employers. If you’re new to Beam, you can begin your learning journey with this quest. To attempt this quest, you don’t need any prior knowledge of data processing or distributed systems. All you need is elementary knowledge about programming.
```

> Individuals aren’t the only ones who can benefit from completing this quest -  organizations can too! Because earning this badge represents deep knowledge of an industry leading big data library, having the badge validates your organization’s understanding of Beam.. In addition, you can run the Beam library on a wide variety of runners, including Google Cloud Dataflow, Flink, Spark, and more,  making knowledge about this library highly transferable. Finally, your organization can use this quest as onboarding material for new hires on big data teams, allowing teams and organizations to get their newest employees up-to-date on the latest and greatest that Apache Beam has to offer.

Reviewer 2: ```suggestion
Individuals aren’t the only ones who can benefit from completing this quest -  organizations can too! Because earning this badge represents deep knowledge of an industry leading big data library, having the badge validates your organization’s understanding of Beam. In addition, you can run the Beam library on a wide variety of runners, including Google Cloud Dataflow, Flink, Spark, and more, making knowledge about this library highly transferable. Finally, your organization can use this quest as onboarding material for new hires on big data teams, allowing teams and organizations to get their newest employees up-to-date on the latest and greatest that Apache Beam has to offer.
```

> Given the recent advancements in artificial intelligence, now’s the time to jump into the world of data processing! Get started on your journey [here](https://www.cloudskillsboost.google/quests/310), whether you’re a long time Beam expert or looking to write your first “wordcount” job.

Reviewer: Let's make this the last paragraph. And let's make "Get started on your journey [here](https://www.cloudskillsboost.google/quests/310)," the last sentence of this paragraph.

> If you have feedback or encounter issues, [reach out](/community/contact-us) or email `support@qwiklabs.com`!

Reviewer: Do we need this sentence?  Support could be left out, qwiklabs has its on support mechanims.

We can can drop this sentence or change to "If you have any feedback, please [reach out](/community/contact-us)."

> Please [reach out](/community/contact-us) or email `support@qwiklabs.com` if you have any feedback or encounter any issues!

Author: @aaltay are you ok to allow users to contact the Beam community if theres an issue?

Reviewer: I made a suggestion, see my comment.

> Please [reach out](/community/contact-us) or email `support@qwiklabs.com` if you have any feedback or encounter any issues!

Reviewer 2: ```suggestion
If you have feedback or encounter issues, [reach out](/community/contact-us) or email `support@qwiklabs.com`!
```
